sg_otio/sg_cut_track_writer.py

```diff
@@ -1121,3 +1123,48 @@ def _create_input_media_version(self, input_media, track_name, sg_project, sg_li
             "sg_uploaded_movie"
         )
         return version, published_file
+
+    @staticmethod
+    def get_local_storage_relative_path(local_storage, file_path):
+        """
+        Given a Local Storage and a file path, returns the relative path
+        to the file from the Local Storage, if any.
+
+        :param file_path: Absolute and normalized path to a directory or a file
+                          as a string, it can be in an abstract form.
+        :param str file_path: The path to a file.
+        :returns: The file's relative path to the local storage path, if any.
+        """
+        # Make sure we use the right separator for the os. On Windows, add the
+        # current drive if there is not one in the given file path.
+        if sys.platform == "win32":
+            file_path = os.path.abspath(os.path.normpath(file_path))
+        else:
+            file_path = file_path.replace("\\", "/")
+        file_path = os.path.normcase(file_path)
+        dir_path = os.path.dirname(os.path.normpath(file_path))
+        file_basename = os.path.basename(file_path)
+        if dir_path != os.sep:
+            # Paths returned by os.path.dirname don't include
+            # a trailing separator, this is why we add one here.
+            # The path is normalized above, so we use os.sep.
+            dir_path = "%s%s" % (dir_path, os.sep)
+        local_path = local_storage["path"]
+        # Special case for "/" or "E:\\" were adding an additional separator
+        # will cause a mismatch
+        if local_path == "/" or re.match(r"[A-Za-z]:\\", local_path):
```

Comment:
Isn't some code missing here? It looks we're only handling cases where the local storage path is "/" or "X:\\"?
May be some indentation problem?

Reply:
It is an indentation problem I introduced after testing, not sure what I did. Anyway, fixed now. I should probably move this method to the `utils.py` module and add some tests, but I didn't want to spend too much time on it, so I just did some basic testing in the interpreter.

Reply:
And you need to add unit test, coverage is down by 0.11%